packages/styles/scss/components/card/index.scss

```diff
@@ -69,12 +69,6 @@
       }
     }
     &__CTA {
-      max-width: carbon--mini-units(40);
-
-      .#{$prefix}--card__copy p {
-        @include carbon--type-style('expressive-heading-02');
```

Comment:
@kennylam I removed the bolding here, so that it matched the card styling. It looks like an old dev added this here and there was no specs of why the CTA copy is bolded

Reply:
Got it, thanks!